thoth/common/openshift.py

```diff
@@ -1136,6 +1142,220 @@ def get_dependency_monkey_template(self):
 
         return self._get_template("template=dependency-monkey")
 
+    def schedule_build_analyse(
+        self,
+        document_id: str,
+        output: str,
+        output_format: str = "json",
```

Comment:
BTW I would hardcode this to OpenShift template, we will not produce anything else than JSON in the cluster.

Reply:
ACK, thanks will include the value of output_format to template and make the changes to files accordingly.